index.html

```diff
@@ -10545,11 +10545,7 @@ <h3>User Prompt Handler</h3>
      <li><p>If the <var>requested prompt handler</var>&apos;s [=prompt
      handler configuration/handler=] is not equal to the <a>user
      prompt handler</a>&apos;s [=prompt handler
-     configuration/handler=], or the <var>requested prompt
-     handler</var>&apos;s [=prompt handler configuration/notify=]
-     is not equal to the <a>user prompt
-     handler</a>&apos;s [=prompt handler configuration/handler=],
-     return false.
+     configuration/handler=], return false.
```

Comment:
So, removing this extra check for `notify` means that for a classic session and multiple `alwaysMatch` fields of the requested capabilities the value for `unhandledPromptBehavior` could vary between `accept/dismiss` and `accept/dismiss and notify` without causing a mismatch, right?

Reply:
It doesn't apply in that case, this is only matching against the current configured value. If you're starting a new session the current configured value is always null.

Reply:
Ah, right. That's perfect then! Thanks.